src/core/modules/Planet.hpp

```diff
@@ -199,12 +199,15 @@ class Planet : public StelObject
 	//! - distance
 	//! - phase (result of getPhase)
 	//! - illumination (=100*phase)
-	//! - phase-angle
-	//! - phase-angle-dms (formatted string)
-	//! - phase-angle-deg (formatted string)
-	//! - elongation
-	//! - elongation-dms (formatted string)
-	//! - elongation-deg (formatted string)
+	//! - phase-angle (radians)
+	//! - phase-angle-dms (formatted string; DMS)
+	//! - phase-angle-deg (formatted string; degrees)
+	//! - elongation (radians)
+	//! - elongation-dms (formatted string; DMS)
+	//! - elongation-deg (formatted string; degrees)
+	//! - modern-elongation (elongation in ecliptic longitude or Δλ; on Earth only; radians)
```

Comment:
I fixed the "modern" name use (changed to ecl-elongation), because it is actually the more classical use.